### DISCUSSION TITLE
using slash commands and buttons

### DISCUSSION THREAD

**Discussion by K0126** [aiko-chan-ai/DiscordBotClient#55] [General]
Any way to use slash commands and buttons?

**Comment by aiko-chan-ai** [comment]
bots cannot use the `/interactions` endpoint to interact with other bots

bots can use some slash commands instead
![image](https://github.com/aiko-chan-ai/DiscordBotClient/assets/71698422/f308048a-389d-420a-bc3e-db8b5220526d)

**Comment by sd355729** [comment]
Can I use Midjourney?

**Reply by Wissididom** [reply]
Afaik Midjourney also uses `/interactions`, so no, it won't work, as it was already told 12th July 2023.
